### PR TITLE
Py client verbose

### PR DESCRIPTION
**Pull request by abidlabs**
Small PR to add a verbose flag to the client and also print a message if a user checks the status of a job and there's a long queue

### CODE REVIEW

File: client/python/gradio_client/client.py [hunk 1]
```diff
@@ -635,15 +643,21 @@ def __init__(
...
         future: Future,
         communicator: Communicator | None = None,
+        verbose: bool = True,
+        space_id: str | None = None,
     ):
         """
         Parameters:
             future: The future object that represents the prediction call, created by the Client.submit() method
-            communicator: The communicator object that is used to communicate between the client and the background thread running the job
+            communicator: The c ommunicator object that is used to communicate between the client and the background thread running the job
```
Comment (freddyaboulton) [inline client/python/gradio_client/client.py:652]: Weird space between `c` and `ommunicator`

File: client/python/gradio_client/client.py [hunk 2]
```diff
@@ -772,6 +786,12 @@ def status(self) -> StatusUpdate:
                 )
             else:
                 with self.communicator.lock:
+                    eta = self.communicator.job.latest_status.eta
+                    if self.verbose and self.space_id and eta and eta > 0:
```
Comment (freddyaboulton) [inline client/python/gradio_client/client.py:790]: I think `eta > 0` is too loose of a condition, no?

**Review by freddyaboulton — APPROVED** [review]
Looks good @abidlabs !

File: client/python/gradio_client/client.py (hunk 2 above)
Comment (abidlabs) [reply client/python/gradio_client/client.py:790]: Oh sorry that was supposed to >30 lol

Comment (abidlabs) [reply client/python/gradio_client/client.py:790]: Changed it during test, will revert

**Comment by abidlabs** [comment]
Thanks for the quick review @freddyaboulton! Fixed these two mistakes

**Pull request merged**